xml/security-troubleshooting_tls.xml

```diff
@@ -102,6 +102,14 @@ ansible-playbook -i hosts/verb_hosts monasca-start.yml</screen>
    Percona clusters.
   </para>
   <procedure>
+   <step>
+    <para>
+      Determine if the TLS certificates for MySQL / Percsona have expired.
+    </para>
+    <screen>&prompt.ardana;cd /etc/mysql/
+&prompt.ardana;openssl x509 -noout -enddate -in control-plane-1-mysql-internal-cert.pem | grep After
```

Comment:
You don't need the `| grep After` part as the `-enddate` option will only print out the expiration date.

Reply:
fixed, thanks! tested out as well just in case
```
ardana-jmoffitt-tls-test-8-cp1-c0-m1-mgmt:~ # ssh ardana-jmoffitt-tls-test-8-cp1-c1-m1-mgmt
ardana-jmoffitt-tls-test-8-cp1-c1-m1-mgmt:~ # cd /etc/mysql/
ardana-jmoffitt-tls-test-8-cp1-c1-m1-mgmt:/etc/mysql # openssl x509 -noout -enddate -in control-plane-1-mysql-internal-cert.pem
notAfter=May  6 15:05:22 2022 GMT
```